frontend/src/components/atoms/PltAmount.vue

```diff
@@ -14,6 +14,7 @@ type Props = {
 	value: string
 	decimals: number
 	formatNumber?: boolean
+	fixedDecimals?: number
```

Comment:
Would be good to add comments what the `decimals` and what the `fixedDecimals` values are (explaining the differences) as people using the component should be able to set them correctly.